build/codegen/codegen.sh

```diff
@@ -14,29 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GROUP_VERSIONS="ceph.rook.io:v1"
+set -o pipefail
+set -o nounset
+set -o errexit
```

Comment:
I requested that these be applied exactly as in the example update codegen script, but the ordering here is reversed. errexit, nounset, pipefail is the standard ordering I see across many scripts. It's also in alphabetical order. There are also maintenance benefits to keeping our script boilerplate components identical to the one its based on.

I want to note that I take great care when I do code reviews to be clear and exact with my feedback. I consistently see that what I request gets modified in some way in the next iteration without seeming reason or purpose. I'm not sure if this is a misunderstanding that it seems like what I'm asking is unspecific, or if it's deliberately contrary. I want to make it clear that more often than not, I am being specific about what change I want to see and not general. When I give generalized feedback, I try to also make sure that is clear.
 
```suggestion
set -o errexit
set -o nounset
set -o pipefail
```